### ISSUE TITLE
Use shortcut keys to save a specific workspace

### ISSUE THREAD

**Issue by ss818** [nothingislost/obsidian-workspaces-plus#66]
When wwitching jobs, I like to save the current workspace so that I can quichly restore it next time. I made several workspaces named "temporary". However, saving the workspace as some "temporary" is still very cumbersome.

I hope to be able to save a certain "temporary" workspace using shortcut.

**Comment by jsmorabito** [comment]
can you share more details, maybe with an example, so I can better understand what you are trying to accomplish?

**Comment by nothingislost** [comment]
I think I get it. Just like we have a shortcut to load a named workspace, they want a shortcut to save a named workspace.

Lets say you load workspace A make a bunch of changes. Then you want to switch to Workspace B temporarily but you don't want to save all of your changes to Workspace A, you just want to switch to Workspace B and then go back to your previous state.

Having a shortcut to save the current state to `Workspace Temp` would allow you to hit the keyboard shortcut, switch to Workspace B, and then revert back by loading `Workspace Temp`

**Comment by jsmorabito** [comment]
@nothingislost that makes sense. would this workflow only work if autosave is turned off then? an extension of this idea the might be useful is to have autosave be toggleble per workspace. A mode called "autosave" that would only toggle the autosave option on and off could even be created to make this workflow even more efficient

**Comment by nothingislost** [comment]
Yes, auto save would need to be off for this to work. Auto save per workspace probably warrants a separate issue.

**Comment by ss818** [comment]
Sorry, I rarely use github and I am very unskilled. You have fully understood what I mean.
